pkgs/development/ocaml-modules/junit/default.nix

```diff
@@ -0,0 +1,22 @@
+{ lib, fetchFromGitHub, buildDunePackage, ptime, tyxml }:
+
+let
+  shared = (import ./shared.nix) { inherit lib fetchFromGitHub; };
+in
+  buildDunePackage rec {
+    pname = "junit";
+    inherit (shared) src version meta;
+    useDune2 = true;
+
+    buildInputs = [
```

Comment:
Only one of either `propagatedBuildInputs` or `buildInputs` is required.

Reply:
Done